app/assets/javascripts/Components/Modals/extension_modal.jsx

```diff
@@ -83,47 +87,54 @@ class ExtensionModal extends React.Component {
     });
   };
 
+  getLabel = (time) => {
+    if (time === 'weeks') {
+      return I18n.t('durations.weeks.other');
+    } else if (time === 'days') {
+      return I18n.t('durations.days.other');
+    } else if (time === 'hours') {
+      return I18n.t('durations.hours.other');
+    } else if (time === 'minutes') {
+      return I18n.t('durations.minutes.other');
+    }
+  };
+
+  renderTimeInput = () => {
+    return this.props.times.map( (time) =>
+      <label key={time}>
+        <input
+          type="number"
+          value={this.state[time]}
+          max={time === 'minutes' ? 60 : 999}
+          min={0}
+          name={time}
+          onChange={this.handleModalInputChange}
+        /> {this.getLabel(time)}
+      </label>
+    )
```

Comment:
https://github.com/pyta-uoft/pyta/pulls

Reply:
Can you explain this please?

Reply:
Whoops sorry, I think I meant to copy-and-paste a comment but didn't copy properly.

There should be a semicolon on this line.